eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -30,6 +31,10 @@ parameters:
                 template: %ezplatform.default_view_templates.content.line%
                 match: []
         embed:
+            image:
+                template: '%ezplatform.default_view_templates.content.embed_image%'
+                match:
+                    IsEmbedImage: true
```

Comment:
We have had that since EZP-25000, so no. My points where the following:

> I fully agree we need to abstract image matching so I get why this is done, but
> - This seems to pollute the xml
> - Class afaik doesn't have any meaning in the kernel, this changes it 
> - What about content upgraded from eZ Publish, it won't contain the class, and it will be hard to know when to apply it actually, cause this is editorial knowledge _(user picks)_ and not system generated knowledge _(based on content type)_  like it was in the past.
> 
> On kernel side from my perspective what was supposed to solve this is Asset Domain types, or just Assets if it was up to CBA. 

And since Assets is not specified in terms of how the mapping is done, doing something prematurely that we will need to cleanup later is not optimal. content type matching is something we support out of the box and people know from legacy and the other matchers.


After a bit back and forward we agreed that if want to make this editorial, assuming PM wants that, then the right place to put it could be `view`, that would make the matching cleaner also.